tests/commands/test_build.py

```diff
@@ -430,6 +432,38 @@ def test_build_code_simple(tmp_path):
     assert linked_entrypoint == built_entrypoint
 
 
+def test_build_code_simple_with_config(tmp_path):
+    """Check transferred metadata, config, and simple entrypoint."""
+    build_dir = tmp_path / BUILD_DIRNAME
+    build_dir.mkdir()
+
+    metadata = tmp_path / CHARM_METADATA
+    entrypoint = tmp_path / 'crazycharm.py'
+    config = tmp_path / CHARM_CONFIG
+    config.touch()
+
+    builder = Builder({
+        'from': tmp_path,
+        'entrypoint': entrypoint,
+        'requirement': [],
+    })
+    linked_entrypoint = builder.handle_code()
+
+    built_metadata = build_dir / CHARM_METADATA
+    assert built_metadata.is_symlink()
+    assert built_metadata.resolve() == metadata
+
+    built_entrypoint = build_dir / 'crazycharm.py'
+    assert built_entrypoint.is_symlink()
+    assert built_entrypoint.resolve() == entrypoint
```

Comment:
Sorry, I should have been more explicit in the last comment: This test should test only that , as it's present, the "config" is included (no need to test again metadata, entry point, etc.).

Thanks!!!